pkg/cmd/pipeline/describe.go

```diff
@@ -63,7 +63,8 @@ func printPipelineDescription(out io.Writer, p cli.Params, pname string) error {
 
 	pipeline, err := cs.Tekton.TektonV1alpha1().Pipelines(p.Namespace()).Get(pname, metav1.GetOptions{})
 	if err != nil {
-		return err
+		fmt.Fprintf(out, emptyMsg)
```

Comment:
I am tempted to use `stderr` to write this message so that `out` is empty

Reply:
@chmouel @vdemeester wdyt?

Reply:
yes make sense to me